SetEnv.targets

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+ 	<Import Project=".\build\SetEnv.tasks"/>
+	<Import Project=".\ReleaseVersion.props"/>
+    <Target Name="SetEnvironmentVariables" Condition="'$(IsOfficialBuild)' != 'True'" BeforeTargets="BeforeBuild">	
```

Comment:
Can we pick a more descriptive name for this target (and this file)? If I understand correctly, it is responsible for overriding the package build number defined in the `project.json`. Something like `OverridePackageBuildNumber` would be more helpful than `SetEnvironmentVariables` to someone reading this code.